### PR TITLE
Singular of dives is dive

### PR DESCRIPTION
**Pull request by mcammaert**
singular inflections for dives / dive.

### CODE REVIEW

File: tests/Doctrine/Tests/DoctrineTestCase.php [hunk 1]
```diff
@@ -2,9 +2,15 @@
 
 namespace Doctrine\Tests;
 
+// backward compatibility
```
Comment (GrahamCampbell) [inline tests/Doctrine/Tests/DoctrineTestCase.php]: 👎

Comment (mcammaert) [inline tests/Doctrine/Tests/DoctrineTestCase.php]: @GrahamCampbell This was an attempt to fix the travis build errors. It was not meant to be included in the pull request.

Comment (mcammaert) [inline tests/Doctrine/Tests/DoctrineTestCase.php]: I reverted these commits

**Comment by alcaeus** [comment]
@mcammaert could you rebase on top of master to fix build failures? Thanks!

**Comment by Ocramius** [comment]
Manually merged via doctrine/inflector@034bfe4600e301aa9bf7ee2e508a88e075295220

**Pull request closed without merging**